bet-predictor/cmd/helper.go

```diff
@@ -0,0 +1,73 @@
+package cmd
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"log"
+	"os"
+	"time"
+
+	"github.com/serhiihuberniuk/bet-predictor/config"
+	fetcher "github.com/serhiihuberniuk/bet-predictor/fetcher/es-fetcher"
+	"github.com/serhiihuberniuk/bet-predictor/repository"
+	"github.com/serhiihuberniuk/bet-predictor/service"
+	"go.mongodb.org/mongo-driver/mongo"
+	"go.mongodb.org/mongo-driver/mongo/options"
+	"go.mongodb.org/mongo-driver/mongo/readpref"
+)
+
+type credentials struct {
+	ApiKey string `json:"api_key"`
+}
+
+func commandInit() (context.Context, *fetcher.ESFetcher, *service.Service, func(), error) {
```

Comment:
better to not return so many params and use structure for that, some `CommandContext`